spec/polaris-catalog-apis/oauth-tokens-api.yaml

```diff
@@ -0,0 +1,99 @@
+#
```

Comment:
Not a big deal but I'd suggest renaming this file to `polaris-token-service.yaml` to align with other files and also with how we call this endpoint generally in Polaris code. Wdyt?

Reply:
I thought the convention is `<some-name>-api.yaml` to refer to APIs in `polaris-catalog-apis` folder, and `polaris-catalog-service.yaml` is outside of that folder is kind of the only exception?